tests/common/helpers.hpp

```diff
@@ -39,6 +39,10 @@
     BOOST_CHECK_EQUAL(posting_auths, POSTING);                     \
 }
 
+// Check if 2 numeric values are approximately equal
+#define APPROX_CHECK_EQUAL(X, Y, DELTA) {                          \
```

Comment:
This define is not needed. 
You should fix calculation in this file: https://github.com/GolosChain/golos/blob/master/tests/common/comment_reward.hpp

Reply:
@afalaleev What should I fix in this file? Could you describe it a bit more detailed?